src/compiler/python_private_generator.h

```diff
@@ -59,6 +59,9 @@ struct PrivateGenerator {
                  const grpc_generator::Service* service,
                  grpc_generator::Printer* out);
 
+  bool PrintServiceClass(const grpc::string& package_qualified_service_name,
+                         const grpc_generator::Service* service,
+                         grpc_generator::Printer* out);
```

Comment:
optional: Should we try to make the generated code PEP8-complied? https://github.com/grpc/grpc/issues/5014
nit: Can we check if linters (`yapf`, `pylint`) are happy with the generated code?

Reply:
That seems reasonable except perhaps for snake case. I've seen people complain in some languages that the casing matches their languages' preferences and I've seen other complain that it doesn't match what they wrote in their ".proto" file. One point against this is that the `protobuf` module is *not* going to do this, so they'd end up mixing snake case with camel case, which IMO, is worse than the situation we currently have.

Reply:
Updated to 4 spaces.